arwen/elrondapi/bigIntOps.go

```diff
@@ -317,7 +331,10 @@ func bigIntStorageStoreUnsigned(context unsafe.Pointer, keyOffset int32, keyLeng
 		return 0
 	}
 
-	value := bigInt.GetOne(source)
+	value, err := bigInt.GetOne(sourceHandle)
+	if arwen.WithFault(err, context, runtime.BigIntAPIErrorShouldFailExecution()) {
```

Comment:
for this case you need GetOrCreate

Reply:
here the handle is given as param through the rust framework function, however i changed it in arwen to GetOneOrCreate(), just to be sure.  
Look like this : 
`fn storage_store_big_uint_raw(&self, key: &[u8], handle: i32) {
		unsafe {
			bigIntStorageStoreUnsigned(key.as_ref().as_ptr(), key.len() as i32, handle);
		}
	}`